### PR TITLE
quality(errorprone with NullAway)

### PR DESCRIPTION
**Pull request by the-pear**
See #8

### CODE REVIEW

File: quality/README.md [hunk 1]
```diff
@@ -67,3 +67,22 @@ jacocoAndroidUnitTestReport {
...
+## Errorprone (with NullAway)
+// root script
+buildscript {
+  apply from: "config/index.gradle"
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
+  }
```
Comment (NemoOudeis) [inline quality/README.md]: have you tried putting it into the `quality/errorprone/android.gradle` using the `plugin` dsl?
sometimes that worked, sometimes it doesn't. would be cleaner if that works.

File: quality/errorprone/android.gradle [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apply plugin: 'net.ltgt.errorprone'
+
+dependencies {
+  annotationProcessor "com.uber.nullaway:nullaway:0.3.7"
```
Comment (NemoOudeis) [inline quality/errorprone/android.gradle]: why not `0.4.0`?

File: quality/README.md (hunk 1 above)
Comment (the-pear) [reply quality/README.md]: I tried plugin first, it's not working.

File: quality/errorprone/android.gradle (hunk 2 above)
Comment (the-pear) [reply quality/errorprone/android.gradle]: OK, latest was 0.3.7 before lunch. Will update.

File: quality/README.md (hunk 1 above)
Comment (NemoOudeis) [reply quality/README.md]: :(

File: quality/errorprone/android.gradle (hunk 2 above)
Comment (NemoOudeis) [reply quality/errorprone/android.gradle]: LOOOOL what a timing xD

**Pull request merged**